app/scripts/model/dataProxy.js

```diff
@@ -3,10 +3,11 @@ window.DataManagerForIviz = (function($, _) {
   var content = {};
   // DESC
   var clinicalAttrsPriority = ['CANCER_TYPE', 'CANCER_TYPE_DETAILED',
-    'GENDER', 'AGE', 'sequenced', 'has_cna_data', 'sample_count_patient'];
+    'GENDER', 'SEX', 'AGE', 'sequenced', 'has_cna_data', 'sample_count_patient'];
```

Comment:
Its gonna be a hard fix to change the code every when ever there is update to this attributes list. We need to find different solution.